python/MSSMvsSM.py

```diff
@@ -252,6 +252,14 @@ def doHistFuncForQQH(self, varlist):
                 br_htautau = getattr(self.mssm_inputs, self.quantity_map['br']['method'])(accesskey_br, x, y)
                 br_htautau_SM = getattr(self.mssm_inputs, self.quantity_map['br_SM']['method'])(accesskey_br_SM, x, y)
 
+                if not br_htautau and not br_htautau_SM:
```

Comment:
What does this `if` clause actually mean?

Reply:
If values are zero ---> not properly filled. This is usual python logic (instead of using `br_htautau != 0`, which is more C++ style.)

Reply:
See the printouts following the clause to understand what is happening.

Reply:
> If values are zero ---> not properly filled. This is usual python logic (instead of using `br_htautau != 0`, which is more C++ style.)

Where did you get this from. In my opinion checking this the value explicitly is way more readable. The printouts were what confused me initially.

Reply:
changed... hope it is now better